src/content/developers/docs/oracles/index.md

```diff
@@ -1,314 +1,86 @@
-- [Generate verifiable random numbers (useful for gaming)](https://chain.link/solutions/chainlink-vrf)
-- [Call external APIs](https://docs.chain.link/docs/request-and-receive-data)
-  – One novel use of this is [Checking wBTC reserves](https://cointelegraph.com/news/1b-in-wrapped-bitcoin-now-being-audited-using-chainlink-s-proof-of-reserve)
-
-Here is an example of how to get the latest ETH price in your smart contract using a Chainlink price feed:
-
-### Chainlink Data Feeds {#chainlink-data-feeds}
-
-```solidity
-pragma solidity ^0.6.7;
-
-import "@chainlink/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol";
-
-contract PriceConsumerV3 {
-
-    AggregatorV3Interface internal priceFeed;
+Giving smart contracts the ability to execute based on real-world inputs vastly extends the value of decentralized applications. For instance, decentralized prediction markets rely on oracles to provide information about outcomes with which they can validate predictions from users. If Alice bets 20 ETH on who becomes the next U.S. President, the prediction-market dapp needs an oracle to confirm election results and determine if Alice is eligible for a payout. 
 
-    /**
-     * Network: Kovan
-     * Aggregator: ETH/USD
-     * Address: 0x9326BFA02ADD2366b30bacB125260Af641031331
-     */
-    constructor() public {
-        priceFeed = AggregatorV3Interface(0x9326BFA02ADD2366b30bacB125260Af641031331);
-    }
-
-    /**
-     * Returns the latest price
-     */
-    function getLatestPrice() public view returns (int) {
-        (
-            uint80 roundID,
-            int price,
-            uint startedAt,
-            uint timeStamp,
-            uint80 answeredInRound
-        ) = priceFeed.latestRoundData();
-        return price;
-    }
-}
-```
+## Prerequisites {#prerequisites}
 
-[You can test this in remix with this link](https://remix.ethereum.org/#version=soljson-v0.6.7+commit.b8d736ae.js&optimize=false&evmVersion=null&gist=0c5928a00094810d2ba01fd8d1083581)
+This page assumes the reader is familiar with Ethereum fundamentals, including [nodes](/developers/docs/nodes-and-clients/), [consensus mechanisms](/developers/docs/consensus-mechanisms/), and the [EVM](/developers/docs/evm/). You should also have a good grasp of [smart contracts](/developers/docs/smart-contracts/) and [smart contract anatomy](/developers/docs/smart-contracts/anatomy/), especially events. 
 
-[View the docs](https://docs.chain.link/docs/get-the-latest-price)
+## What is a blockchain oracle? {#what-is-a-blockchain-oracle} 
 
-### Chainlink VRF {#chainlink-vrf}
+Oracles are applications that source, verify, and transmit external information (i.e., information stored off-chain) to smart contracts running on the blockchain. Besides “pulling” off-chain data and broadcasting it on Ethereum, oracles can also “push” information from the blockchain to external systems. An example is an oracle that unlocks a smart lock once the user sends the fee via an Ethereum transaction. 
```

Comment:
```suggestion
Oracles are applications that source, verify, and transmit external information (i.e. information stored off-chain) to smart contracts running on the blockchain. Besides “pulling” off-chain data and broadcasting it on Ethereum, oracles can also “push” information from the blockchain to external systems. An example is an oracle that unlocks a smart lock once the user sends the fee via an Ethereum transaction.
```

Reply:
> An example is an oracle that unlocks a smart lock once the user sends the fee via an Ethereum transaction.

Unclear what this is an example of, it could be:

a) an oracle
b) pulling information
c) pushing information